src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/git/dialog/GitReviewPanel.css

```diff
@@ -152,4 +152,13 @@
 
 .ignoreWhitespace label {
 	margin-left: 4px;
+}
+
+.charCountPanel {
```

Comment:
For consistency, use `.visuallyHidden` from themeStyles.css. There are a few different techniques for this, might as well be using the same one everywhere.

Reply:
Done!